src/main/java/rx/Completable.java

```diff
@@ -1843,7 +1846,7 @@ public final Completable startWith(Completable other) {
      */
     public final Subscription subscribe() {
         final MultipleAssignmentSubscription mad = new MultipleAssignmentSubscription();
-        subscribe(new CompletableSubscriber() {
+        unsafeSubscribe(new CompletableSubscriber() {
```

Comment:
But the idea of the safe subscribe is to ensure the caller that the contract will be met. Why in this particular case it won't?